cnnimg.py

```diff
@@ -65,111 +62,107 @@ def imageprocessing(self, inputlocation, outputlocation, tempA, tempB, initialco
         u = np.array(gray)
         u = u[:,:,0]
         z0 = (u)*initialcondition
-        self.Bu = sig.convolve2d(u, tempB, 'same')
-        self.tempA = tempA
-        self.Ib = Ib
-        z0 = z0.reshape(z0.size)
-        z = self.cnn(sint.odeint(self.f, z0, t))
+        Bu = sig.convolve2d(u, tempB, 'same')
+        z0 = z0.flatten()
```

Comment:
Was there any improvement by using this? It didn't break any functionality anywhere, did it?